ckan-backend-dev/src/ckanext-wri/ckanext/wri/schema/ckan_dataset.yaml

```diff
@@ -0,0 +1,185 @@
+  preset: tag_string_autocomplete
+  form_placeholder: eg. economy, mental health, government
+
+- field_name: temporal_coverage
+  label: Temporal Coverage
+  validators: iso_datetime_range
+
+- field_name: update_frequency
+  label: Update Frequency
+
+- field_name: citation
+  label: Citation
+  form_snippet: markdown.html
+  form_placeholder: eg. Citation information
+
+- field_name: visibility_type
+  label: Visibility Type
+  preset: select
+  choices:
+  - value: draft
+    label: Draft
+  - value: private
+    label: Private
+  - value: internal
+    label: Internal
+  - value: public
+    label: Public
+
+- field_name: license_id
+  label: License
+  form_snippet: license.html
+  help_text: License definitions and additional information can be found at http://opendefinition.org/
+
+- field_name: featured_dataset
+  label: Featured Dataset
+  validators: boolean_validator
+
+- field_name: short_description
+  label: Short Description
+  form_snippet: markdown.html
+  form_placeholder: eg. A short description of the dataset
+
+- field_name: notes
+  label: Description
+  form_snippet: markdown.html
+  form_placeholder: eg. Some useful notes about the data
+
+- field_name: author
+  label: Author
+  form_placeholder: Joe Bloggs
+  display_property: dc:creator
+
+- field_name: author_email
+  label: Author Email
+  form_placeholder: joe@example.com
+  display_property: dc:creator
+  display_snippet: email.html
+  display_email_name_field: author
+
+- field_name: maintainer
+  label: Maintainer
+  form_placeholder: Joe Bloggs
+  display_property: dc:contributor
+
+- field_name: maintainer_email
+  label: Maintainer Email
+  form_placeholder: joe@example.com
+  display_property: dc:contributor
+  display_snippet: email.html
+  display_email_name_field: maintainer
+
+- field_name: function
+  label: Function
+  form_placeholder: eg. This data is used to...
+
+- field_name: restrictions
+  label: Restrictions
+  form_placeholder: eg. This data is restricted to...
+
+- field_name: reason_for_adding
+  label: Reason for Adding
+  form_placeholder: eg. This data is being added because...
+
+- field_name: learn_more
+  label: Learn More
+  form_placeholder: eg. https://example.com/learn_more.pdf
+  validators: url_validator
+
+- field_name: cautions
+  label: Cautions
+  form_placeholder: eg. This data should be used with caution because...
+
+- field_name: summary
+  label: Summary
+  form_snippet: markdown.html
+  form_placeholder: eg. A short summary of the dataset
+
+resource_fields:
+
+- field_name: url
+  label: URL
+  preset: resource_url_upload
+
+- field_name: name
+  label: Name
+  form_placeholder: eg. January 2011 Gold Prices
+
+- field_name: description
+  label: Description
+  form_snippet: markdown.html
+  form_placeholder: Some useful notes about the data
+
+- field_name: format
+  label: Format
+  preset: resource_format_autocomplete
+
+- field_name: resource_type
+  label: Type of data
+
+- field_name: resource_tiles
```

Comment:
I think you should probably remove all of those related to tha maps, they are going to be either stored in the views object or just in the RW API, you could put however a field that is something like "rw_id" and "gfw_id" so that we could trace it back to their original apis

Reply:
Yes, cause these are all related to the layer, so if they are going to appear somewhere is either in the views or in the external apis